modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/appcatalog/ComputeResourceFileSystemEntity.java

```diff
@@ -33,11 +30,21 @@
  */
 @Entity
 @Table(name="compute_resource_file_system")
```

Comment:
Maybe upper case the table name here too, for consistency?

Reply:
@machristie Thanks for the review. 
Made the change